src/components/home/chat-bubble/components/__tests__/image-dialog.test.tsx

```
@@ -0,0 +1,52 @@
+import { render, screen } from "@testing-library/react";
+import { vi } from "vitest";
+import { ImageDialog } from "../image-dialog";
+
+// Mock Next.js Image component
+vi.mock("next/image", () => ({
+  default: ({ src, alt, className, fill }: any) => (
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)